packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -811,6 +814,7 @@ export const tagData = <
 		// eslint-disable-next-line unicorn/no-array-reduce, unicorn/prefer-object-from-entries
 		.reduce((pv, cv) => {
 			const [key, value] = cv;
+			// eslint-disable-next-line unicorn/prefer-ternary
```

Comment:
Just curious: why do we not prefer the ternary here?

Reply:
Because it's already nested inside a reduce call, the ternary expression is less readable than this. I'll add a comment.

Reply:
```suggestion
			// The ternary form is less legible in this case.
			// eslint-disable-next-line unicorn/prefer-ternary
```